JapaneseFilter/sections/css_extended.txt

```diff
@@ -475,7 +475,7 @@ libae.jp#?#.post > div.pc_display > div[id^="logly"]:upward(1)
 hibiki-radio.jp#?#.ng-isolate-scope > li.ng-scope ins.adsbygoogle:upward(li)
 geartics.com#?#.mb-3.text-center:contains(スポンサーリンク)
 trailrunner.jp#?#.news_list > li > ins.adsbygoogle:upward(1)
-kabumatome.doorblog.jp#?#blockquote:has(a[href^="https://www.amazon.co."])
+otakomu.jp,kabumatome.doorblog.jp#?#blockquote:has(a[href^="https://www.amazon.co."])
```

Comment:
It has false positive: `http://otakomu.jp/archives/30470970.html`
Please change to a new rule `otakomu.jp#?#blockquote:last-of-type:has([href*="amazon.co."])`

Reply:
ok